### ISSUE TITLE
Health Sector Actors Mapping

### ISSUE THREAD

**Issue by SamuelPetragallo** [WorldHealthOrganization/herams-backend#235]
We discussed developing an specific extract of HeRAMS data that would provide the list of partners and their related locations (GEO1) and activities (ownership/management/support) as a means of extrapolating a service provision 3W

Let's discuss its structure and how technically it could be served (right now we mostly have open text fields for partners name which we may need to change (realize that's quite some work + means admin then need to be locally trained on maintaining this list - not sure it's a easy pick for now)

**Comment by SamMousa** [comment]
Is this still relevant?

**Comment by SamuelPetragallo** [comment]
Yes, though not urgent. This kind of analysis is a standard in humanitarian response management when lots of partners are involved in the response and coordination key. We have the data but not being able to spit out the analysis easily, means parallel processes being implemented at the country level to get the same data. Below an example @luna404 has developed for Mali. 

I don't think that packaging the info should be too much of an issue. Guess it will also be a lot about where we offer it to users (how visible and practical we make it). This is likely to generate a lot of interest beyond regular HeRAMS users.

**Comment by luna404** [comment]
I think we first need to find another way than open text fields to get partner names. I had more than 10 different ways users spelled the same partner. Any automated analysis you do on this kind of data will be almost impossible to interpret.

**Comment by SamuelPetragallo** [comment]
If we had it, we could start offering it for project where this consolidation has been done? (and hence have a good argument to realign others)
